import-service/serverless.yml

```diff
@@ -0,0 +1,53 @@
+service:
+  name: import-service
+# app and org for use with dashboard.serverless.com
+#app: your-app-name
+#org: your-org-name
+frameworkVersion: "2"
+
+# Add the serverless-webpack plugin
+plugins:
+  - serverless-webpack
+  - serverless-offline
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+  stage: dev
+  region: eu-west-1
+
+  iamRoleStatements:
+    - Effect: "Allow"
+      Action: "s3:ListBucket"
+      Resource:
+        - "arn:aws:s3:::node-aws-product-imports"
+    - Effect: "Allow"
+      Action:
+        - "s3:*"
+      Resource:
+        - "arn:aws:s3:::node-aws-product-imports/*"
+
+custom:
+  webpack:
+    webpackConfig: ./webpack.config.js
+    includeModules:
+      forceExclude:
+        - aws-sdk
+
+functions:
+  importProductsFile:
+    handler: src/lambdas/importProductsFile.get
+    events:
+      - http:
+          method: get
+          path: import
```

Comment:
Please add queryString parameters to the serverless.yml. There is one of the requirements